bson/bsonoptions/string_codec_options.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package bsonoptions
+
+// StringCodecOptions represents all possible options for time.Time encoding and decoding.
+type StringCodecOptions struct {
+	ObjectIDAsHex *bool // Specifies if we should decode ObjectID as the hex value. Defaults to true.
+}
+
+// StringCodec creates a new *StringCodecOptions
+func StringCodec() *StringCodecOptions {
+	return &StringCodecOptions{}
+}
+
+// SetObjectIDAsHex specifies if we should decode ObjectID as the hex value. Defaults to true.
+func (t *StringCodecOptions) SetObjectIDAsHex(b bool) *StringCodecOptions {
```

Comment:
Can we change the name to `DecodeObjectIDAsHex`? I think good rule of thumb for these names should be to start with `Encode`/`Decode` for clarity.

Reply:
done